corgi/core/models.py

```diff
@@ -847,7 +847,7 @@ def get_purl(self) -> PackageURL:
                 qualifiers["epoch"] = str(self.epoch)
             purl_data = dict(
                 name=self.name,
-                version=f"{self.version}-{self.release}",
+                version=f"{self.version}{f'-{self.release}' if self.release else ''}",
```

Comment:
f-string nesting is a bit iffy in my opinion. It's not used anywhere in the Python docs and feels a bit less readable then just separating it out into an if statement before the dict is created. Do you prefer this one-line form?

Reply:
Ok, I removed nested f-stings. I saw it already existed in the code so I just reused it as it looked succinct.